files/aldap.py

```diff
@@ -71,7 +71,10 @@ def __decode__(self, word:bytes) -> str:
 
 	def __findMatch__(self, group:str, adGroup:str):
 		# Extract the Common Name from the string (letters, spaces, underscores and hyphens)
-		adGroup = re.match('(?i)CN=((\w*\s?_?-?)*)', adGroup).group(1)
+		try:
+			adGroup = re.match('(?i)CN=((\w*\s?_?-?)*)', adGroup).group(1)
+		except AttributeError:
+			None
```

Comment:
maybe we should `return None` ?

Reply:
Ack yeah that'd make sense. I've updated the commit :)